src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -65,6 +73,25 @@ var fluid_2_0 = fluid_2_0 || {};
             },
             "onPanelShow.operate": {
                 listener: "{that}.operateShow"
+            },
+            "onCreate.setAriaStates": "{that}.setAriaStates"
+        },
+        members: {
+            panelId: {
+                expander: {
+                    // create an id for panel
+                    // and set that.panelId to the id value
+                    funcName: "fluid.allocateSimpleId",
+                    args: "{that}.dom.panel"
+                }
+            }
+        },
+        model: {
+            isShowing: false
+        },
+        modelListeners: {
+            "{that}.model.isShowing": {
+                funcName: "{that}.setAriaStates"
```

Comment:
You ave a setAriaStates call as a handler for onCreate, so you could either remove it (if it doesn't actually have to happen onCreate, or you should set the modelListener here to `excludeSource: "init"` so that it isn't run on initialization.